App doesn't start in Docker

### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

The Docker version of Directus fails to start. To me it seems like that `sanitize-html` was added to `devDependencies` instead of `dependencies` by mistake in this commit:
https://github.com/directus/directus/commit/25375cc4810b1e501cce1590ad8c21148ea596cb#diff-d0ab9c1cc78fba971b5270314ad877706d340d099b42f46a406fc5de6a32fdf0

### To Reproduce

docker run \
  -p 8055:8055 \
  -e KEY=255d861b-5ea1-5996-9aa3-922530ec40b1 \
  -e SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263 \
  directus/directus:9.1.0

### Errors Shown

`Error: Cannot find module 'sanitize-html'
Require stack:
- /directus/node_modules/directus/dist/utils/md.js
- /directus/node_modules/directus/dist/services/notifications.js
- /directus/node_modules/directus/dist/services/activity.js
- /directus/node_modules/directus/dist/services/index.js
- /directus/node_modules/directus/dist/controllers/activity.js
- /directus/node_modules/directus/dist/app.js
- /directus/node_modules/directus/dist/server.js
- /directus/node_modules/directus/dist/cli/index.js
- /directus/node_modules/directus/dist/cli/run.js
- /directus/node_modules/directus/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/directus/node_modules/directus/dist/utils/md.js:8:41)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/directus/node_modules/directus/dist/utils/md.js',
    '/directus/node_modules/directus/dist/services/notifications.js',
    '/directus/node_modules/directus/dist/services/activity.js',
    '/directus/node_modules/directus/dist/services/index.js',
    '/directus/node_modules/directus/dist/controllers/activity.js',
    '/directus/node_modules/directus/dist/app.js',
    '/directus/node_modules/directus/dist/server.js',
    '/directus/node_modules/directus/dist/cli/index.js',
    '/directus/node_modules/directus/dist/cli/run.js',
    '/directus/node_modules/directus/cli.js'
  ]
}
`

### What version of Directus are you using?

9.1.0

### What version of Node.js are you using?

The one that the Docker version uses by default

### What database are you using?

The one that the Docker version uses by default

### What browser are you using?

-

### What operating system are you using?

macOS

### How are you deploying Directus?

Docker

--- a/api/package.json
+++ b/api/package.json
@@ -146,6 +146,7 @@
 		"rate-limiter-flexible": "^2.2.2",
 		"resolve-cwd": "^3.0.0",
 		"rollup": "^2.52.1",
+		"sanitize-html": "^2.6.0",
 		"sharp": "^0.29.0",
 		"stream-json": "^1.7.1",
 		"supertest": "^6.1.6",
@@ -208,7 +209,6 @@
 		"copyfiles": "2.4.1",
 		"cross-env": "7.0.3",
 		"jest": "27.3.1",
-		"sanitize-html": "^2.5.3",
 		"ts-jest": "27.0.7",
 		"ts-node-dev": "1.1.8",
 		"typescript": "4.5.2"

